generators/client/templates/angular/src/main/webapp/app/admin/logs/logs.component.html.ejs

```diff
@@ -26,8 +26,8 @@
     <table class="table table-sm table-striped table-bordered" aria-describedby="logs-page-heading">
         <thead>
         <tr title="click to order">
-            <th (click)="orderProp = 'name'; reverse=!reverse"><span jhiTranslate="logs.table.name">Name</span></th>
-            <th (click)="orderProp = 'level'; reverse=!reverse"><span jhiTranslate="logs.table.level">Level</span></th>
+            <th scope="row" (click)="orderProp = 'name'; reverse=!reverse"><span jhiTranslate="logs.table.name">Name</span></th>
```

Comment:
shouldn't scope be `col` ?

Reply:
:man_facepalming: Yes of course, obviously it was too late yesterday